src/fosslight_util/download.py

```
@@ -680,19 +714,207 @@ def get_github_token(git_url):
+        return False, (
+            f"Download aborted: temporary directory size exceeded "
+            f"limit ({float(size_limit_gb):g} GB) after {elapsed_hint}."
+        )
+
+    stdout = stderr = ""
+    try:
+        next_timeout = size_check_after_sec
+        first_check_done = False
+        while True:
+            try:
+                stdout, stderr = proc.communicate(timeout=next_timeout)
```

Ruff (0.15.21)

[warning] Unpacked variable `stdout` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)